package.json

```diff
@@ -24,7 +24,7 @@
     "src/*"
   ],
   "dependencies": {
-    "react-native-linear-gradient": "https://raw.githubusercontent.com/wordpress-mobile/react-native-linear-gradient/v2.5.6-wp-3/react-native-linear-gradient-2.5.6-wp-3.tgz",
+    "react-native-linear-gradient": "https://raw.githubusercontent.com/wordpress-mobile/react-native-linear-gradient/update/android-13/react-native-linear-gradient-2.5.6-wp-4.tgz",
```

Comment:
This reference will change once https://github.com/wordpress-mobile/react-native-linear-gradient/pull/8 is merged and a tag is generated.

Reply:
Done in [Bump react-native-linear-gradient to point tag version](https://github.com/wordpress-mobile/react-native-hsv-color-picker/pull/9/commits/b3235c158158635d4b620ab2d179e955287723bc).